tests/functional/u/use/use_dict_literal.py

```diff
@@ -0,0 +1,7 @@
+# pylint: disable=missing-docstring, invalid-name
+
+x = dict()  # [use-dict-literal]
```

Comment:
Can you rename the test files?
```
use_dict_literal -> use_literal_dict
use_list_literal -> use_literal_list
```

That will make testing them a bit easier, as you then can do `pytest tests/test_functional -k use_literal` to just test these two.

Reply:
Maybe ``-k literal`` is close enough ? :)

Reply:
I only renamed the test files. Should I have changed the message code as well? To keep it in line with document names? Or is this fine?

Reply:
@cdce8p Just making sure you didn't miss this comment. No changes needed?

Reply:
The comment was just about the test file names. So everything is fine with it